pkg/adapter/config/kubernetes/doguConfigRepository.go

```diff
@@ -0,0 +1,38 @@
+package kubernetes
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-38 lines are duplicate of `pkg/adapter/config/kubernetes/sensitiveDoguConfigRepository.go:1-38`
<!-- __reviewdog__:ChAxMDFjOWRjZjQzZmFlN2UxEgpjaGVja3N0eWxl -->